### PR TITLE
DIS-95 Add upvote count to post domain object and response groups

### PR DESCRIPTION
**Pull request by jogura**
Exposes the upvote count for posts in the responses.

* Update the db schema to add the `upvote_count` to the `POST` table
* Update SQL to use new query
* Update domain object to include `upvotes` field
* Update mappers to include field
* Update tests to test for new field

@jimmordino @armonr

### CODE REVIEW

**Comment by armonr** [comment]
:shipit:

File: service/discussion/src/test/java/com/wikia/discussionservice/resources/PostResourceIntegrationTest.java [hunk 1]
```diff
@@ -110,10 +110,10 @@ public void postReplyToThread() {
     assertThat(newPost.getRawContent(), equalTo(postBody));
 
     testThread = getForumThread(SITE_ID_GLEE, TEST_REPLY_THREAD_ID);
-    assertThat(testThread.getPostCount(), equalTo(new Integer(originalThreadPostCount + 1)));
+    assertThat(testThread.getPostCount(), equalTo(originalThreadPostCount + 1));
     assertThat(testThread.getLastPostId(), equalTo(newPost.getId()));
     testForum = getForum(SITE_ID_GLEE, testThread.getForumId());
-    assertThat(testForum.getPostCount(), equalTo(new Integer(originalForumPostCount + 1)));
+    assertThat(testForum.getPostCount(), equalTo(originalForumPostCount + 1));
   }
```
Comment (jimmordino) [inline service/discussion/src/test/java/com/wikia/discussionservice/resources/PostResourceIntegrationTest.java]: FYI, I don't like to use auto-boxing.  Explicit is better than implicit, even in Java.

**Comment by jimmordino** [comment]
LGTM

File: service/discussion/src/test/java/com/wikia/discussionservice/resources/PostResourceIntegrationTest.java (hunk 1 above)
Comment (jogura) [inline service/discussion/src/test/java/com/wikia/discussionservice/resources/PostResourceIntegrationTest.java]: That's one of the warnings from our static checks. 'Unnecessary auto-boxing'. I changed it to the faster `Integer.valueOf()` and it still complained.

**Pull request merged**